### DISCUSSION TITLE
Is there a way to group duplicated articles?

### DISCUSSION THREAD

**Discussion by czlowieknikt** [martinrotter/rssguard#990] [Q&A]
When having multiple feeds providing the same article (e.g. sharing info from other services) is there a way to group them and displaying as one (with ability to expand with the detailed source list)?

**Comment by martinrotter** [comment]
Checkout "article filtering" feature which can do exactly what you want.

https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#fltr

```js
function filterMessage() {
  if (msg.isAlreadyInDatabase(MessageObject.SameTitle | MessageObject.AllFeedsSameAccount)) {
    return MessageObject.Purge;
  }
  else {
    return MessageObject.Accept;
  }
}
```
